frontend/src/components/StaticTokenKubeconfigSwitch.vue

```diff
@@ -0,0 +1,45 @@
+<!--
+SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+<template>
+  <div>
+    <v-checkbox
+      v-model="enableStaticTokenKubeconfig"
+      label="Enable static token kubeconfig"
```

Comment:
```suggestion
      label="Enable static token kubeconfig (not recommended)"
```
strange, I thought @holgerkoser  and I already made this suggestion, but somehow it got lost

Reply:
okay